src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -7078,4 +7178,51 @@ export class StudyViewPageStore {
 
         this.setPatientTreatmentFilters({ filters: updatedFilters });
     }
+
+    @computed get mutationFilterActive() {
```

Comment:
would it make sense to prefix these with "global" ?

Reply:
I did this and some other refactorings (use of @computed rather than function calls, made a functional component form a function, name changes)